lib/fetch.js

```diff
@@ -125,16 +126,19 @@ export async function fetchEventById(eventId) {
 
 export async function getSpeakersByEventId(eventId) {
   const api = await Prismic.api(PrismicConfig.apiEndpoint);
-  const res = await api.form('everything').ref(api.master()).query(
-    Prismic.Predicates.at('document.type', 'speaker'),
-    Prismic.Predicates.fulltext('my.speaker.eventId', eventId)
-  )
-  .submit();
-
-  if (res && res.results.length > 0) {
-    return res.results.map(sanitizeSpeaker);
+  try {
+    const res = await api.form('everything').ref(api.master()).query(
+      Prismic.Predicates.at('document.type', 'speaker'),
+      Prismic.Predicates.has('my.speaker.event', eventId)
+    )
+    .submit();
+    if (res && res.results.length > 0) {
+      return res.results.map(sanitizeSpeaker);
+    }
+  } catch (e) {
+    return [];
```

Comment:
I don't think we should be suppressing exceptions like this. They should be propagated up the call stack and handled accordingly - i.e in middleware